src/tx/builder/schema.js

```diff
@@ -22,7 +22,8 @@ export const RESPONSE_TTL = { type: 'delta', value: 10 }
 // # CONTRACT
 export const DEPOSIT = 0
 export const AMOUNT = 0
-export const GAS = 25000
+export const GAS_ABOVE_AVERAGE = 25000 // TODO: don't use this
```

Comment:
`GAS_ABOVE_AVERAGE` is confusing. Do you mean this is the default Gas limit used for methods not supported by dry-run?

Reply:
> Do you mean this is the default Gas limit used for methods not supported by dry-run?

Yep. In particular, this is the gas value that is above the average gas of contract calls/deployments. I was using it in `createGeneralizeAccount` but seems that it is not needed there.